### PR TITLE
setup mon repo on mgr nodes when rhcs cdn debian config is set

### PR DESCRIPTION
**Pull request by vasukulkarni**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1594974

Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

File: roles/ceph-common/tasks/installs/prerequisite_rhcs_cdn_install_debian.yml [hunk 1]
```diff
@@ -13,5 +13,5 @@
   with_items:
     - { repo: "MON", configure: (mon_group_name in group_names) }
     - { repo: "OSD", configure: (osd_group_name in group_names) }
-    - { repo: "Tools", configure: (rgw_group_name in group_names or mds_group_name in group_names) }
+    - { repo: "Tools", configure: (rgw_group_name in group_names or mds_group_name in group_names or mgr_group_name in group_names) }
```
Comment (leseb) [inline roles/ceph-common/tasks/installs/prerequisite_rhcs_cdn_install_debian.yml]: Add `nfs_group_name in group_names or iscsi_gw_group_name in group_names or client_group_name in group_names` too.

Comment (leseb) [reply roles/ceph-common/tasks/installs/prerequisite_rhcs_cdn_install_debian.yml]: Also, edit https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/tasks/installs/prerequisite_rhcs_cdn_install.yml#L15 to add `mgr_group_name in group_names`. Thanks.

Comment (vasukulkarni) [reply roles/ceph-common/tasks/installs/prerequisite_rhcs_cdn_install_debian.yml]: @leseb sure will make changes and repush, thanks!

**Comment by vasukulkarni** [comment]
@leseb to be consistent, I am using mon repo for mgr and toosl for nfs/iscsi/clients.

**Comment by leseb** [comment]
@vasukulkarni looks good now thanks!

**Pull request merged**